src/components/Common/Footer/Footer.jsx

```diff
@@ -0,0 +1,9 @@
+import style from './Footer.module.css';
+
+export function Footer() {
+  return (
+    <footer className={style.footer}>
+      <p>CopyRight: Evgeniy Moroz, 2022</p>
```

Comment:
Create method for it and cover with unit tests. Year can be changed, name can be also passed as a parameter and in general everything should be flexible, but not hard-coded

Reply:
moved to constants

```suggestion
        {FOOTER_KEYS.copyRight} {FOOTER_KEYS.years}
```